src/components/MTableGroupbar/index.js

```diff
@@ -32,6 +32,26 @@ function MTableGroupbar(props) {
     borderStyle: 'dashed'
   });
 
+  useEffect(() => {
+    if (props.persistentGroupingsId) {
+      const persistentGroupings = [];
+
+      props.groupColumns.forEach((column) => {
+        persistentGroupings.push({
+          field: column.field,
+          groupOrder: column.tableData.groupOrder,
+          groupSort: column.tableData.groupSort,
+          columnOrder: column.tableData.columnOrder
+        });
+      });
+
+      localStorage.setItem(
+        props.persistentGroupingsId,
```

Comment:
Do we want to pretag that with material table or something to reduce friction with other possible items?

Reply:
I aggregate the groupings under material-table-groupings now